### PR TITLE
[develop] Feature/metric update

### PR DESCRIPTION
**Pull request by clouden90**
## DESCRIPTION OF CHANGES: 
Update the WE2E test used in `srw_metric_example.sh` to reflect the changes that the WE2E test "grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16" has been substituted with "grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_WoFS_v0" in #686 (The GFS v16 suite is not intended for use at such high resolutions, so using the WoFS_v0 suite was more appropriate in this case).  

## TESTS CONDUCTED: 
To start, git clone the feature branch, then navigate to the ufs-srweather-app folder and proceed to check out the external repositories. Set the following environment variables: WORKSPACE (path to your `ufs-srweather-app` folder), SRW_PLATFORM (e.g., `hera`), SRW_COMPILER (e.g., `intel`), and SRW_PROJECT (e.g., `nems`). Once these are set, run the command `./.cicd/scripts/srw_metric_example.sh`. This process is expected to take around 1 and a half hours to complete. Afterward, you should find an output file named skill-score.out that contains the skill score index. Tests have been conducted on Hera and [Jenkins pipeline](https://jenkins.epic.oarcloud.noaa.gov/job/sandbox/job/SRW_metric_example_pipeline/16/console).

- [x] hera.intel
- [ ] orion.intel
- [ ] cheyenne.intel
- [ ] cheyenne.gnu
- [ ] gaea.intel
- [ ] jet.intel
- [ ] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [x] Jenkins
- [ ] fundamental test suite
- [ ] comprehensive tests (specify *which* if a subset was used)

## CHECKLIST

- [x] My code follows the style guidelines in the Contributor's Guide
- [x] I have performed a self-review of my own code using the Code Reviewer's Guide
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes do not require updates to the documentation (explain).
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

## LABELS (optional): 

A Code Manager needs to add the following labels to this PR: 
- [ ] Work In Progress
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] release
- [ ] high priority
- [ ] run_ci
- [ ] run_we2e_fundamental_tests
- [ ] run_we2e_comprehensive_tests
- [ ] Needs Cheyenne test 
- [ ] Needs Jet test 
- [ ] Needs Hera test 
- [ ] Needs Orion test 
- [ ] help wanted

### CODE REVIEW

**Review by natalie-perlin — APPROVED** [review]

File: tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_WoFS_v0.yaml [hunk 1]
```diff
@@ -12,6 +14,15 @@ workflow:
...
   PREEXISTING_DIR_METHOD: rename
+rocoto:
+  tasks:
+    taskgroups: '{{ ["parm/wflow/prep.yaml", "parm/wflow/coldstart.yaml", "parm/wflow/post.yaml", "parm/wflow/verify_pre.yaml", "parm/wflow/verify_det.yaml"]|include }}'
+    metatask_run_ensemble:
+      task_run_fcst_mem#mem#:
+        walltime: 01:00:00
+    task_get_obs_ccpa:
+    task_get_obs_mrms:
+    task_get_obs_ndas:
```
Comment (MichaelLueken) [inline tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_WoFS_v0.yaml]: Please remove the empty `task_get_obs_*` entries.  The changes in Mike's PR #864, which was recently merged, will check to see if data is currently staged before attempting to pull data from HPSS.  These empty tasks are no longer required and were removed in PR #864.

**Review by MichaelLueken — COMMENTED** [review]
@clouden90 -

With the latest develop HEAD, the empty `task_get_obs_*` entries are no longer required.  If the files are staged, then the steps will be skipped.  With the removal of these entries, I was able to successfully run the updated forecast metric test without issue.  Once these have been removed, I will be ready to approve this work.

File: tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_WoFS_v0.yaml (hunk 1 above)
Comment (clouden90) [reply tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_WoFS_v0.yaml]: Done

**Review by MichaelLueken — APPROVED** [review]
Thanks, @clouden90!  Approving now.

**Review by mkavulich — APPROVED** [review]
Thanks @clouden90 for getting this turned around so quickly! I'm not familiar enough with the STATAnalysisConfig file to review those changes, but the rest looks good to me.

**Comment by MichaelLueken** [comment]
The Cheyenne Intel tests were manually run on Hera and passed:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
custom_GFDLgrid__GFDLgrid_USE_NUM_CELLS_IN_FILENAMES_eq_FALSE      COMPLETE               7.65
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot     COMPLETE              29.78
grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16                COMPLETE              19.61
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_HRRR           COMPLETE              25.49
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta    COMPLETE               8.72
grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_HRRR_suite_HRRR                COMPLETE              16.69
pregen_grid_orog_sfc_climo                                         COMPLETE               7.46
specify_template_filenames                                         COMPLETE               7.88
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE             123.28
```
The Cheyenne GNU tests were manually run on Hera and passed:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
grid_CONUS_25km_GFDLgrid_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16      COMPLETE              20.45
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta      COMPLETE             227.87
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v17_p8_plot  COMPLETE              27.59
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR             COMPLETE              36.70
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP              COMPLETE              81.58
grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16   COMPLETE              23.08
grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP                 COMPLETE             319.60
grid_SUBCONUS_Ind_3km_ics_NAM_lbcs_NAM_suite_GFS_v16               COMPLETE              51.59
MET_verification_only_vx                                           COMPLETE               0.12
specify_EXTRN_MDL_SYSBASEDIR_ICS_LBCS                              COMPLETE              10.36
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE             798.94
```
The manual test of MET_ensemble_verification_only_vx_time_lag on Hera GNU passed:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
MET_ensemble_verification_only_vx_time_lag                         COMPLETE               7.66
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE               7.66
```
The manual testing of the WE2E coverage tests on Jet (including custom_ESGgrid_Great_Lakes_snow_8km) have passed:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
community                                                          COMPLETE              18.71
custom_ESGgrid                                                     COMPLETE              18.53
custom_ESGgrid_Great_Lakes_snow_8km                                COMPLETE              19.11
custom_GFDLgrid                                                    COMPLETE              10.04
get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_nemsio_2021032018         COMPLETE              13.12
get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_netcdf_2022060112_48h     COMPLETE              60.19
get_from_HPSS_ics_RAP_lbcs_RAP                                     COMPLETE              17.25
grid_RRFS_AK_3km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR                 COMPLETE             241.34
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot     COMPLETE              41.43
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2        COMPLETE               8.61
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta       COMPLETE             525.13
nco_grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR       COMPLETE              11.63
process_obs                                                        COMPLETE               0.30
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE             985.39
```
The Jenkins tests successfully passed on Gaea, Hera, and Orion, so I will now move forward with merging these changes.

**Pull request merged**